app/serializers/api/v2/serializable_accrediting_provider.rb

```diff
@@ -0,0 +1,13 @@
+# This lightweight serializer is to reduce the amount of database
+# hits needed to build this and reduce the amount of data sent
+# over the wire that isn't used in the front-end
+
+module API
+  module V2
+    class SerializableAccreditingProvider < JSONAPI::Serializable::Resource
+      type 'accrediting_providers'
+
+      attributes :provider_code, :provider_name
```

Comment:
Don't we need the description here too?

Reply:
That's what we discussed yesterday, I stripped it way back to build upon it. This should perhaps be a draft PR.